### PR TITLE
responsive for desktop

### PR DESCRIPTION
**Pull request by thecodingvivek**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,140 +1,253 @@
...
+                <svg class="sandwichsvg" viewBox="0 0 20 20" width="22">
+                    <rect class="line top" width="20px" height="3px" y="3" rx="1.5"></rect>
+                    <rect class="line middle" width="20px" height="3px" y="9" rx="1.5"></rect>
+                    <rect class="line bottom" width="20px" height="3px" y="15" rx="1.5"></rect>
+                </svg>
+            </button>
+
+            <div class="mobile-nav">
+              <nav>
+                <a class="navbaractive" onclick="setTimeout(()=> {closenav()},200);">Map <span class="material-symbols-outlined leftmargin5">map</span></a>
```
Comment (Abhinav-ark) [inline index.html:27]: Why do we need a delay here?

Comment (Abhinav-ark) [reply index.html:27]: The delay ensures that the hamburger animation and hiding of the navbar are synchronized with each other.

**Review by Abhinav-ark — CHANGES_REQUESTED** [review]
## Navigator is not displaying next destination in mobile view.
https://github.com/user-attachments/assets/ebb1c055-0323-4966-a6d0-74c6822ab603

The navigator which shows directions, next destination and distance is not displaying the later in mobile view.

**Review by Abhinav-ark — APPROVED** [review]
Good For Now.

**Review by ROHITH-M10 — APPROVED** [review]
LGTM.

**Pull request merged**